src/components/ButtonPanel/button-panel.component.jsx

```diff
@@ -14,7 +16,22 @@ const ButtonPanel = () => {
     <div className="groups">
       { Object.keys(groups).map((group, i) => (
         <div key={keys[i]} className={`group ${group}`}>
-          {groups[group].map((name, j) => <Button key={keys[j]} name={name} />)}
+          {groups[group]
+            .map((name, j) => {
+              let title;
+              let buttonName = name;
+              if (name.includes(',')) {
+                [buttonName, title] = name.split(',');
+              }
+              return (
+                <Button
+                  clickHandler={handleClick(props)}
```

Comment:
**[Required]**
Let us validate handleClick as well. All props have to be validated.